store/sqlstore/upgrade.go

```diff
@@ -1353,3 +1355,46 @@ func upgradeDatabaseToVersion600(sqlStore *SqlStore) {
 	// saveSchemaVersion(sqlStore, Version600)
 	// }
 }
+
+func upgradeDatabaseToVersion610(sqlStore *SqlStore) {
+	// if shouldPerformUpgrade(sqlStore, Version600, Version610) {
+
+	forceIndex := ""
+	if sqlStore.DriverName() == model.DatabaseDriverMysql {
+		forceIndex = "FORCE INDEX(idx_posts_channel_id)"
```

Comment:
Starting in 5.37 we don't have this index any longer as it was redundant. We should use `idx_posts_channel_id_update_at` for now.